dynamic/coinchange_test.go

```diff
@@ -0,0 +1,30 @@
+package dynamic
```

Comment:
Use black-box testing:
```suggestion
package dynamic_test
```

Reply:
if i use dynamic_test, the output is
./coinchange_test.go:23:14: undefined: CoinChange
FAIL	github.com/TheAlgorithms/Go/dynamic [build failed]

how do i run code, if i use black-box testing?

Reply:
You first need to change the function name to export it.